src/features/academic/components/progress/AreaProgressSection.module.scss

```
@@ -6,18 +6,13 @@
-
+// 섹션 헤더 — '전체 수강내역'·'필수 이수내역' 과 동일 스타일(body-md·bold·gray-500·여백)로 통일.
 .dualMajorTitle {
-  @include typography.body-lg;
+  @include typography.body-md;
   @include typography.font-weight-bold;
-  margin-bottom: spacing.$space-12;
-}
-
-.dualMajorSpacing {
-  margin-bottom: spacing.$space-20;
+  margin-top: spacing.$space-28;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)